Makefile

```diff
@@ -228,6 +228,10 @@ version:
 	@echo "FRAMEWORK_BUILD_VERSION:" ${FRAMEWORK_BUILD_VERSION}
 	@echo "XDG_DATA_HOME:" $(XDG_DATA_HOME)
 
+.PHONY: upload-daily-build
+upload-daily-build:
+	cd ./hack/dailybuild && $(MAKE) run && cd ../..
```

Comment:
We could use `pushd` and `popd` too here or `cd -` instead of `cd ../..` which is usually tricky when directory structures change and we miss out to change it